kivy_ios/recipes/numpy/__init__.py

```diff
@@ -11,7 +11,7 @@ class NumpyRecipe(CythonRecipe):
     libraries = ["libnpymath.a", "libnpyrandom.a"]
     include_dir = "numpy/core/include"
     depends = ["python"]
-    hostpython_prerequisites = ["Cython"]
+    hostpython_prerequisites = ["Cython==0.29.17"]
```

Comment:
Don't we support lower than <

Reply:
I pulled the version for Cython from requirements.txt in the root of the repo, if there's a different version that should be locked to such as the latest build from v0.29 (0.29.36) that would make sense to me.

Reply:
On CI pipeline we use `Cython==0.29.33`, `requirements.txt` have `Cython==0.29.17`, on `setup.cfg` is unpinned, and on `hostpython_prerequisites` is also unpinned.

@jswillard what about pinning `0.29.36` everywhere? (for now, see: https://github.com/kivy/buildozer/pull/1637 for full story)

CI fails on this PR as `numpy` complains about the following:
`RuntimeError: Building NumPy requires Cython >= 0.29.30`